addon/services/intl.js

```diff
@@ -6,6 +6,7 @@
 import Ember from 'ember';
 import { IntlMessageFormat } from '../utils/data';
 import createFormatCache from '../format-cache/memoizer';
+import computedPolyfill from 'ember-new-computed';
```

Comment:
call it `computed` and replace all instances of Ember.computed with ember-new-computed :+1: